schema/filter_artworks.js

```diff
@@ -186,6 +216,7 @@ function filterArtworks(primaryKey) {
       }
 
       return gravity.with(accessToken)("filter/artworks", gravityOptions)
+        .then(response => assign({}, response, { options: gravityOptions }))
```

Comment:
It's not obvious to me why this is needed? As in, the filter results are now returning the API parameters used for that filter, in an `options` hash on the response?